Adding Lesson 08 & 09 Assignments

KyleCreek opened the pull request.

KyleCreek committed:
Committing Read Me Text

KyleCreek committed:
Adding the News Files to STudent Folder

KyleCreek committed:
'Updating Completed Lesson 01 Assignments'

KyleCreek committed:
Adding Assignment02 Files

KyleCreek committed:
Adding Resultant .JSON File

KyleCreek committed:
Adding Lesson03 Files

KyleCreek committed:
Adding Lesson04 Assignments Due

KyleCreek committed:
Adding Lesson05 Assignment

KyleCreek committed:
Adding lesson06 Assignment Files

ObjectiveJoe commented on students/KyleCreek/lesson06/assignment/src/data_generator.py:
```diff
@@ -0,0 +1,144 @@
+
+    sequence = 0
+    data_list = []
+    while sequence < data_point:
```
Did you consider using a data iterator here? In what ways do you think one would effect your code?

KyleCreek committed:
Adding Lesson07 Code. Includes series interaction with the Mongo Database and the
parallel procesing. Files associated with Lesson07 for grading are "series1.py" and
"parallel_1.py".

KyleCreek committed:
Adding First REvision of "inventory.py" for Assignment08
